Source/WebCore/PAL/pal/cg/CoreGraphicsSoftLink.cpp

```diff
@@ -39,4 +39,6 @@ SOFT_LINK_FRAMEWORK_FOR_SOURCE_WITH_EXPORT(PAL, CoreGraphics, PAL_EXPORT)
 SOFT_LINK_FUNCTION_FOR_SOURCE_WITH_EXPORT(PAL, CoreGraphics, CGContextSetOwnerIdentity, void, (CGContextRef context, task_id_token_t owner), (context, owner), PAL_EXPORT)
 #endif
 
+SOFT_LINK_FUNCTION_MAY_FAIL_FOR_SOURCE_WITH_EXPORT(PAL, CoreGraphics, CGWindowListCreateImage, CGImageRef, (CGRect screenBounds, CGWindowListOption listOption, CGWindowID windowID, CGWindowImageOption imageOption), (screenBounds, listOption, windowID, imageOption), PAL_EXPORT)
```

Comment:
shouldn't this be defined only on Mac? 

Seems to be a Mac only API
https://developer.apple.com/documentation/coregraphics/1454852-cgwindowlistcreateimage

Reply:
Oh, you mean it should actually compile on other ports ;-)